src/app/tests/suites/certification/Test_TC_S_2_6.yaml

```diff
@@ -358,7 +357,7 @@ tests:
           set to 20000 (20s) and no extension field sets."
       PICS: S.S.C00.Rsp
       verification: |
-          ./chip-tool scenesmanagement add-scene 0x0000 0x01 20000 scene1 [] 1 1
+          scenesmanagement add-scene 0x0000 0x01 20000 scene1 [] 1 1
```

Comment:
This change will apply to all steps from Test step 3a onwards.

Reply:
Updated